Dockerfile

```diff
@@ -1,5 +1,6 @@
 ARG DOCKER_ARCH
 FROM $DOCKER_ARCH/amazonlinux:2 as builder
+RUN yum update -y
 RUN yum group install -y "Development Tools"
 RUN yum install -y glibc-static
```

Comment:
It looks like this layer is only used for building `sheltie`, and we actually already do a `yum update -y` for the final layer.